lnwire/custom.go

```diff
@@ -20,10 +69,12 @@ type Custom struct {
 // interface.
 var _ Message = (*Custom)(nil)
 
-// NewCustom instanties a new custom message.
+// NewCustom instantiates a new custom message.
 func NewCustom(msgType MessageType, data []byte) (*Custom, error) {
-	if msgType < CustomTypeStart {
-		return nil, errors.New("msg type not in custom range")
+	if msgType < CustomTypeStart && !IsCustomOverride(msgType) {
+		return nil, fmt.Errorf("msg type: %v not in custom range: %v "+
+			"and not overridden: %v", msgType, CustomTypeStart,
+			CustomTypeOverride)
```

Comment:
So this causes any custom types not defined in `lnwire/messages.go` to be displayed here as `<unknown>`. But since we are definitely going to always have types not defined yet, should we maybe print out the type numbers here rather?

Reply:
iiuc, you mean use `%d` rather than `%v` in the error? 
If yes, updated! If no, not sure what you mean here.

Reply:
yeah that is what I meant :)